specification/cognitiveservices/HealthInsights/healthinsights.oncophenotype/route.oncophenotype.tsp

```diff
@@ -21,7 +22,8 @@ interface OncoPhenotype {
     "./examples/SuccessfulOncoPhenotypeResponse.json",
     "SuccessfulOncoPhenotypeGetAnalyzeStatus"
   )
-  getJob is Azure.Core.ResourceRead<OncoPhenotypeResult>;
+  getJob is StandardOperations.ResourceRead<OncoPhenotypeResult>;
```

Comment:
We should not have any of the changes to use STandardOperations in this PR.  This was a removed breaking change